### PR TITLE
 For #27671 - Search metrics will now be sent in baseline ping

### PR DESCRIPTION
**Pull request by perrymcmanis144**
### Pull Request checklist

- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

Tests: this counter is already sent in metrics & is covered by existing tests
Screenshots: no visible changes
Accessibility: does not impact any user facing feature

### QA

- [x] **QA Needed**

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-debug` task.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation

Used by GitHub Actions.

### CODE REVIEW

File: app/metrics.yaml [hunk 1]
```diff
@@ -6072,7 +6088,8 @@ browser.search:
       - android-probes@mozilla.com
       - erichards@mozilla.com
     expires: never
-
+    no_lint:
+      - BASELINE_PING
```
Comment (Dexterp37) [inline app/metrics.yaml:6092]:
```suggestion
      - BASELINE_PING

```

Let's restore the newline after this.

**Review by Dexterp37 — APPROVED** [review]
r+wc

**Review by rocketsroger — APPROVED** [review]

**Comment by jonalmeida** [comment]
Manually rebasing/merging because it will require us do that either way and we're second in queue.

**Comment by rocketsroger** [comment]
@perrymcmanis144 Merged to Beta v107

**Pull request merged**